src/commands/gemini.ts

```diff
@@ -0,0 +1,31 @@
+import { cli } from "../cli.js";
+import prettier from "../plugins/prettier.js";
+import { unminify } from "../unminify.js";
+import babel from "../plugins/babel/babel.js";
+import { verbose } from "../verbose.js";
+import { geminiRename } from "../plugins/gemini-rename.js";
+import { env } from "../env.js";
+
+export const azure = cli()
+  .name("gemini")
+  .description("Use Google Gemini/AIStudio API to unminify code")
+  .option("-m, --model <model>", "The model to use", "gemini-1.5-flash")
```

Comment:
I wonder if this could be made a 'flexible enum' or similarly provide a list of other useful model names that could be used here in it's help/similar? (while not limiting the model chosen to a fixed list so it doesn't need to be updated for every new model release)